tests/unit/test_prestoclient.py

```diff
@@ -45,11 +45,13 @@ def test_no_user(self, mock_presto_config):
                                 "Username missing",
                                 client.run_sql, "any_sql")
 
+    @patch.object(PrestoClient, '_create_auth_headers', return_value={'X-Presto-Internal-Bearer': 'any_bearer'})
```

Comment:
The problem with this is that we're not exercising the token generation code. We can keep this test as is but re-introduce the `test_create_authorization_headers` test, make `_create_auth_headers` a static method that takes in a user and a secret. We would need to patch `datetime.now()` and it should work.

Or to keep the API clean, we don't need to make `_create_auth_headers` static and just create a request but not provide a secret to force us to generate a token. Patching datetime will be needed then as well.

What do you think?